content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -88,66 +102,114 @@ Events:
   1m           1m          1        {replicaset-controller }             Normal      SuccessfulCreate  Created pod: frontend-qhloh
   1m           1m          1        {replicaset-controller }             Normal      SuccessfulCreate  Created pod: frontend-dnjpy
   1m           1m          1        {replicaset-controller }             Normal      SuccessfulCreate  Created pod: frontend-9si5l
-$ kubectl get pods
+```
+
+And lastly checking on the pods brought up:
+```shell
+kubectl get pods
+```
+
+Will yield pod information similar to
+```shell
 NAME             READY     STATUS    RESTARTS   AGE
 frontend-9si5l   1/1       Running   0          1m
 frontend-dnjpy   1/1       Running   0          1m
 frontend-qhloh   1/1       Running   0          1m
 ```
 
-## Writing a ReplicaSet Spec
+## Non-Template Acquisitions
 
-As with all other Kubernetes API objects, a ReplicaSet needs the `apiVersion`, `kind`, and `metadata` fields.  For
-general information about working with manifests, see [object management using kubectl](/docs/concepts/overview/object-management-kubectl/overview/).
+A ReplicaSet is not limited to owning pods specified by its template. Take the previous frontend ReplicaSet example,
+and the Pods specified in the following manifest:
 
-A ReplicaSet also needs a [`.spec` section](https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status).
+{{< codenew file="pods/pod-rs.yaml" >}}
 
-### Pod Template
+As those Pods do not have a Controller (or any object) as their owner reference and match the selector of the forntend
+ReplicaSet, they will immediately be acquired by it.
+
+Creating the Pods after the frontend ReplicaSet has been deployed and set up its initial Pod replicas to fulfill its
```

Comment:
Splitting this sentence between with a command in the middle doesn't work for me. I got lost trying to parse the first portion of the sentence.

Suggestion: Suppose the frontend ReplicaSet is already deployed, and then you enter this command to create the Pods:
kubectl create ...
The ReplicaSet will immediately acquire the Pods and terminate them, because the ReplicaSet already has its three Pods.

Fetch the Pods:
kubectl get pods

The output shows ...